### PR TITLE
Add switchbot cloud vacuum support

### PR DESCRIPTION
**Pull request by Gigatrappeur**
## Proposed change

Modify documentation for mention vacuum support in switchbot cloud integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125457
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added support for vacuum devices in the SwitchBot Cloud integration.
	- New vacuum models supported: Vacuum K10+, K10+ pro, S1, S1 Plus.
- **Documentation**
	- Updated integration documentation to reflect the latest changes and device compatibility.
	- Release version updated to '2023.10'.

### CODE REVIEW

File: source/_integrations/switchbot_cloud.markdown [hunk 1]
```diff
@@ -41,6 +44,7 @@ Please note, device names configured in the SwitchBot app are transferred into H
 - Meter
 - MeterPlus
 - Outdoor Meter
+- Vacuum K10+, K10+ pro, S1, S1 Plus
```
Comment (c0ffeeca7) [inline source/_integrations/switchbot_cloud.markdown:48]: Please add a section ## Entities ### Vacuum, and list all the supported actions, like Neato does, for example.

**Comment by c0ffeeca7** [comment]
I'm [drafting this PR](https://developers.home-assistant.io/docs/review-process#prs-are-being-drafted-when-changes-are-needed), as changes are needed. Please undraft it, once the changes have been implemented and the PR is ready for review again.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Gigatrappeur :+1:

**Comment by Gigatrappeur** [comment]
Thank you, but you asked me to modify the doc. Is this really necessary?

**Comment by c0ffeeca7** [comment]
> Thank you, but you asked me to modify the doc. Is this really necessary?

It would make it clearer which ones are supported. Can be in a follow-up PR. I merged it now because tomorrow is beta.

**Comment by Gigatrappeur** [comment]
ok. I'm preparing another PR to use webhook with switchbot cloud. So, I'll modify the doc at the same time

**Pull request merged**